File: src/main/java/org/lvtinger/paladin/account/repo/AccountRepository.java
package org.lvtinger.paladin.account.repo;

import org.lvtinger.paladin.account.api.entity.Account;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface AccountRepository extends JpaRepository<Account, Long> {
    boolean existsAccountByUsernameAndDisable(String username, Boolean disable);
    Account getAccountByUsernameAndPassword(String username, String password);
}

File: src/main/kotlin/org/risesun/paladin/provider/impl/repository/AccountRepository.kt
package org.risesun.paladin.provider.impl.repository

import org.risesun.paladin.provider.api.entity.Account
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface AccountRepository : JpaRepository<Account, Long> {
    fun findByUsername(username: String): Account?
}